Create an ACM policy set for Advanced Cluster Security Secured Clusters (#438)
open-cluster-management-io/policy-collection: policygenerator/policy-sets/community/acs-secure/README.md

# Advanced Cluster Security PolicySet for Secured Clusters

## Prerequisites
To install Advanced Cluster Security Secured Clusters using this PolicySet, 
you must first have already installed your Advanced Cluster Security Central Server.
The policies make the following assumptions:

 - RHACS is installed on the RHACM hub and the ACS init bundle secrets are created
   in the `stackrox` namespace.
 - RHACS is installed on the RHACM hub and the ACS Central Server `Route` resource 
   exists in the `stackrox` namespace.
 - An install of Red Hat Advanced Cluster Management for Kubernetes version 2.8
   or newer is required.
  

## Installation

> The ACS PolicySet for Secured Clusters contains two `PolicySets` that will be deployed.i

Reviewer: ```suggestion
The ACS PolicySet for Secured Clusters contains two `PolicySets` that will be deployed.
```

The `PolicySets` install RHACS Secured Clusters onto all OpenShift clusters that are 
managed by RHACM except for the RHACM hub cluster.  If you want to install the RHACS
Secured Cluster component to the RHACM hub, that must be done separately.

Prior to applying the `PolicySet`, perform these steps:

> 1. To allow for subscriptions to be applied below you must apply and set to enforce the policy [policy-configure-subscription-admin-hub.yaml](https://github.com/open-cluster-management-io/policy-collection/blob/main/community/CM-Configuration-Management/policy-configure-subscription-admin-hub.yaml).

Reviewer: This only applies if using App Sub, so maybe this is not a required step.

> 2. Install the Policy generator Kustomize plugin by following the [installation instructions](https://github.com/stolostron/policy-generator-plugin#installation). It is recommended to use Kustomize v4.5+.

Reviewer: ```suggestion
2. Install the Policy generator Kustomize plugin by following the [installation instructions](https://github.com/open-cluster-management-io/policy-generator-plugin#installation). It is recommended to use Kustomize v4.5+.
```

> 3. Policies are installed to the `policies` namespace.  i

Reviewer: ```suggestion
3. Policies are installed to the `policies` namespace. 
```

   Make sure the placement bindings match this namespace for the hub and other managed clusters.
   Example yaml to apply a ManagedClusterSetBinding for the policies namespace.

>     ```apiVersion: cluster.open-cluster-management.io/v1beta2

Reviewer: ```suggestion
    ```
    apiVersion: cluster.open-cluster-management.io/v1beta2
```

# Advanced Cluster Security PolicySet for Secured Clusters

## Prerequisites

> To install Advanced Cluster Security Secured Clusters using this PolicySet, 

Reviewer: ```suggestion

To install Advanced Cluster Security Secured Clusters using this PolicySet, 
```

> you must first have already installed your Advanced Cluster Security Central Server.

Reviewer: ```suggestion
you must have already installed your Advanced Cluster Security Central Server.
```
